src/ppplib.erl

```diff
@@ -1,26 +0,0 @@
--module(ppplib).
```

Comment:
Perhaps it makes sense to leave this module in the library, so that there is one module that unites the API with some additional check for expected arguments and their types. The call of API `ppplib:hdlc_encapsulate/1` looks more attractive instead of `ppplib_hdlc:decapsulate/1.

Reply:
the call would actually be `ppp_hdlc:decapsulate/1`. To me that makes more sense then the `lib` in there